src/sql/workbench/contrib/query/browser/query.contribution.ts

```diff
@@ -335,8 +336,8 @@ const queryEditorConfiguration: IConfigurationNode = {
 		},
 		'queryEditor.results.preferProvidersCopyHandler': {
 			'type': 'boolean',
-			'description': localize('queryEditor.results.preferProvidersCopyHandler', "Whether the copy result request should be handled by the query provider when it is supported. The default value is true, set this to false to force all copy handling to be done by Azure Data Studio."),
-			'default': true
+			'description': localize('queryEditor.results.preferProvidersCopyHandler', "Whether the copy result request should be handled by the query provider when it is supported. The default value is true for Windows and Mac, and false for Linux, set this to false to force all copy handling to be done by Azure Data Studio."),
+			'default': platform.isLinux ? false : true
```

Comment:
Did you get a chance to test this change on a linux machine?

Reply:
I did and my changes didn't introduce any regressions to Linux.